clients/admin-ui/src/types/api/index.ts

```diff
@@ -2,6 +2,13 @@
 /* tslint:disable */
 /* eslint-disable */
 
+export type { AccessToken } from "./models/AccessToken";
```

Comment:
Added additional open api generated typescript models to support the bulk retry API endpoint.